backend/inspirehep/config.py

```diff
@@ -34,7 +34,7 @@
 
 # Web services and APIs
 # =====================
-AUTHENTICATION_TOKEN = "CHANGE_ME"
+AUTHENTICATION_TOKEN = "Yh2LqaWL8Ge1HBgat9H8HR3a6HXNSYtjDpLKDqno3KU5WC74j8L3al6f71up"
```

Comment:
What token is this? Should it be here?

Reply:
it's a leftover, but token is invalid anyhow. I'll change it.